src/Serilog.Sinks.AzureDataExplorer.Samples/Serilog.Sinks.AzureDataExplorer.Samples.csproj

```diff
@@ -8,11 +8,15 @@
   </PropertyGroup>
 
     <ItemGroup>
-      <PackageReference Include="serilog" Version="2.11.0" />
-      <PackageReference Include="Serilog.Sinks.Console" Version="4.0.1" />
```

Comment:
Similarly in the core see if there are updates to use ?

![image](https://user-images.githubusercontent.com/106139410/217153138-2c4d75fd-8b3f-4479-b01b-0d54d5fadb90.png)

Reply:
all other package versions are dev version. Hence not upgraded